src/Form/Input.mdx

```diff
@@ -49,57 +50,79 @@ Form input component. Different sizes are available.
+  <Input multiline placeholder="Example" label="Example" value="I already have a value! If provided a value, I become controlled, so I won't update when you type here..." />
 </Playground>
 
 ### Autogrow Multiline Input
 <Playground>
-   <Input multiline autogrow placeholder="Example" label="Example" />
+  <Input multiline autogrow placeholder="Example" label="Example" />
 </Playground>
 
 ### Floating Input
 <Playground>
-   <Input floating placeholder="Example" label="Example" />
+  <Input floating placeholder="Example" label="Example" />
 </Playground>
 
 ### Disabled Input
 <Playground>
-   <Input disabled placeholder="Example" label="Example" />
+  <Input disabled placeholder="Example" label="Example" />
+</Playground>
+
+### Input with Icons
+<Playground>
+  <Input
+    placeholder="Example"
+    label="Example"
+    leftIcon="cards-spade"
+    onLeftIconClick={()=> alert('Left icon clicked!')}
+    rightIcon="close-circle"
+    rightIconColor="primary"
+    onRightIconClick={()=> alert("Right icon clicked!")}
+  />
 </Playground>
 
 ### Overridable Styles
 <Playground>
-   <Input placeholder="Example" label="Example" styles={{
-     Input: {
-       background: 'beige'
-     }
-   }} />
-
-   <Input floating placeholder="Example" label="Example" styles={{
-     Input: {
-       background: 'beige'
-     }
-   }} />
+  <Input placeholder="Example" label="Example" styles={{
+    Input: {
+      background: 'beige'
+    }
+  }} />
+
+  <Input floating placeholder="Example" label="Example" styles={{
+    Input: {
+      background: 'beige'
+    }
+  }} />
 </Playground>
 
 ### Sizes
 <Playground>
-   <Input size="xs" placeholder="Example" label="Example" />
-   <Input size="sm" placeholder="Example" label="Example" />
-   <Input size="md" placeholder="Example" label="Example" />
-   <Input size="lg" placeholder="Example" label="Example" />
-   <Input size="xl" placeholder="Example" label="Example" />
+  <Input size="xs" placeholder="Example" label="Example" />
+  <Input size="sm" placeholder="Example" label="Example" />
+  <Input size="md" placeholder="Example" label="Example" />
+  <Input size="lg" placeholder="Example" label="Example" />
+  <Input size="xl" placeholder="Example" label="Example" />
 </Playground>
 
 ## Formatted Inputs
 
 ### Phone Input
-<PhoneInput placeholder="Example" label="Phone Number" value="4088675309" />
+<Playground>
+  <PhoneInput placeholder="Example" label="Phone Number" value="4088675309" />
+</Playground>
 
 ### Date Input
-<DateInput placeholder="Date of Birth" label="Date of Birth" initialValue="04/01/2001" value="2001-04-01" />
+<Playground>
+  <DateInput placeholder="Date of Birth" label="Date of Birth" initialValue="04/01/2001" value="2001-04-01" />
+</Playground>
+
+### Search Input
+<Playground>
+  <SearchInput placeholder="Search" label="Search" />
+</Playground>
```

Comment:
I'm not seeing this example locally, is that just me?

Reply:
You might need to run `npm run docs` again? I had the issue early on using `yarn`.